extensions/amp-a4a/0.1/callout-vendors.js

```diff
@@ -42,6 +42,11 @@ export const RTC_VENDORS = {
     macros: ['CID'],
     disableKeyAppend: true,
   },
+  prebid_appnexus : {
```

Comment:
If you're going to do a name with underscores, you'll have to quote the string. You have some lint violations you need to fix as well, but otherwise looks good